pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -80,6 +83,14 @@ class ModelCheckpoint(Callback):
         ...     filepath='my/path/sample-mnist_{epoch:02d}-{val_loss:.2f}'
         ... )
 
+        # retrieve the best checkpoint after training
+        >>> checkpoint_callback = ModelCheckpoint(filepath='my/path/')
+        >>> trainer = Trainer(checkpoint_callback=checkpoint_callback)
+        >>> # model = ...
```

Comment:
what is the purpose here, the model and fir is commented

Reply:
To better show how it's used without actually instantiating a model. If there's a mockup for a `LightningModule`, we can use it, but I couldn't find any.

Reply:
sure but then show is just as code example not as doc test... just remove `>>>` for these 5 lines (this block, keep the above)